chaco/tests/test_array_or_none.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import warnings
+
+import numpy as np
+
+from chaco.array_data_source import ArrayDataSource
+from chaco.axis import PlotAxis
+from chaco.data_label import DataLabel
+from chaco.data_range_1d import DataRange1D
+from chaco.label_axis import LabelAxis
+from chaco.legend import Legend
+from chaco.linear_mapper import LinearMapper
+from chaco.scatterplot import ScatterPlot
+
+
+class TestArrayOrNone(unittest.TestCase):
+    """ Test that the FutureWarning from numpy concerning comparison of arrays
```

Comment:
It would be helpful to expand upon this to include the numpy version where this is a warning and the version where it will become an exception.